PointOfInteraction/SmartPointOfInteraction/doc/spec.md

```diff
@@ -0,0 +1,73 @@
+# SmartPointOfInteraction
+
+## Description
+
+A Smart Point of Interaction defines a place with technology to interact with users, for example, through Beacon technology from Apple, Eddystone/Physical-Web from Google or other proximity-based interfaces. Since the interactive area could be composed by more than one device providing the technology, this model represents a group of Smart Spot devices.
+
+The data model includes information regarding the area/surface covered by the technology (i.e., the area covered by Bluetooth Low Energy-based Beacon), a way to specify the functionality intervals (i.e. when interactive points are available) and the link to the multimedia resource where users will interact (i.e. Web Apps, ...). Additionally, the data model may reference to another NGSI entity such as a Parking, a Point of Interest (POI), etc. with enriched interaction provided by this Smart Point of Interest.
+
+
+This entity is purely virtual, is not a device mapping.
```

Comment:
Which part of the sentence cause confusion? About the reference to other NGSI entity?